trucks.js

```diff
@@ -503,9 +505,55 @@ var foodTrucks = [
 	}
 ];
 
-// this module should support the following methods:
 // getTrucks() - return all trucks
+function getTrucks() {
+	return foodTrucks;
+}
+
 // getTruck(name) - return the truck object matching 'name'
+function getTruck(name) {
```

Comment:
This function would be better just returning the truck object as an object rather than as an array with just one object in it.